src/main.rs

```diff
@@ -78,11 +81,14 @@ fn main() -> Result<()> {
     // Try to enable BPF stats
     let fd = unsafe { bpf_enable_stats(libbpf_sys::BPF_STATS_RUN_TIME) };
     if fd < 0 {
-        return Err(anyhow!("Failed to enable BPF stats"));
+        if let Err(err) = fs::write(BPF_STATS_ENABLED, b"1") {
+            return Err(anyhow!("Failed to enable BPF stats, maybe no BPF_STATS_RUN_TIME or {}", err));
```

Comment:
This error message is confusing. It references an internal constant and that user will not understand.